### DISCUSSION TITLE
(papercut) Request for migration

### DISCUSSION THREAD

**Discussion by AdmiringWorm** [chocolatey-community/chocolatey-packages#2471] [Migrations]
### Why should the package be migrated to this Chocolatey Community Chocolatey Packages Repository?

This is an important package both for the community and Chocolatey, and since this package is now an RFM request over at https://github.com/chocolatey-community/chocolatey-package-requests/issues/1525.

I will take over maintenance of the package first, then, if approved, migrate this package over to this repository.

### What is the link to the package hosted on Chocolatey Community Repository?

https://community.chocolatey.org/packages/papercut

### What is the link to the package source?

https://github.com/ChangemakerStudios/Papercut-SMTP/tree/develop/chocolatey

### Responsibilities

- [X] I will be responsible for creating a PR to migrate the package.
- [X] I will be available to fix any issues with the package or the automatic updater!

**Comment by gep13** [comment]
@AdmiringWorm this sounds like a great idea to me!

Thanks you helping with this!
